test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala

```diff
@@ -0,0 +1,178 @@
+          }
+
+        }
+
+        "the contact preference is Paper" should {
+          val viewModel = ContactNumbersChangeSuccessViewModel(None, Some(ContactPreference.paper), None)
+          lazy val view = injectedView(viewModel)(request, messages, mockConfig, User("1111111111"))
+          lazy implicit val document: Document = Jsoup.parse(view.body)
+
+          "have the correct first paragraph" in {
+            elementText(Selectors.paragraphOne) shouldBe "We will send a letter to your principal place of" +
+              " business with an update within 15 working days."
+          }
+        }
+      }
+
+      "the contact preference is not retrieved" should {
+        val viewModel = ContactNumbersChangeSuccessViewModel(None, None, None)
+        lazy implicit val document: Document = Jsoup.parse(injectedView(viewModel)(request, messages, mockConfig, User("1111111111")).body)
+
+        "have the correct first paragraph" in {
+          elementText(Selectors.paragraphOne) shouldBe "We will send you an update within 15 working days."
+        }
+      }
+    }
+
+    "an agent is performing the action" when {
+
+      "the agent has an email address registered" should {
+        val viewModel = ContactNumbersChangeSuccessViewModel(Some("TheBusiness"), None, Some("agent@example.com"))
+        lazy val view = injectedView(viewModel)(request, messages, mockConfig, User("1111111111", arn = Some(BaseTestConstants.arn)))
+
+        lazy implicit val document: Document = Jsoup.parse(view.body)
+
+        "have the correct page title" in {
+          elementText(Selectors.title) shouldBe "We have received the request to change your telephone numbers"
+        }
+
+        "have the correct heading" in {
+          elementText(Selectors.pageHeading) shouldBe "We have received the request to change your telephone numbers"
+        }
+
+        "have a finish button with the correct text" in {
+          elementText(Selectors.button) shouldBe "Finish"
+        }
+
+        "have a finish button which navigates to the Change of Circs overview page" in {
+          element(Selectors.button).select("a").attr("href") shouldBe "mockManageVatOverviewUrl"
+        }
+
+        "have the correct first paragraph" in {
+          elementText(Selectors.paragraphOne) shouldBe "We‘ll send an email to agent@example.com within 2" +
+            " working days telling you whether or not the request has been accepted."
+        }
+
+        "have the correct second paragraph" in {
+          elementText(Selectors.paragraphTwo) shouldBe "We‘ll also contact TheBusiness with an update."
+        }
+
+        "have a GA tag for the clicking finish button" in {
+          element(Selectors.button).select("h1").attr("data-journey") contains "email-address:confirm:finish-email-change"
+        }
+      }
+
+      "the agent doesn't have an email address registered" should {
```

Comment:
Could you add in an extra test case just for the small content change when the agent's client's business name is not retrieved

Reply:
Ahh assumed the helper was unit tested so wouldn't need to. On it now